src/app/classify.rs

```diff
@@ -171,6 +171,18 @@ impl classify::ClassifyEos for Eos {
     fn error(self, err: &h2::Error) -> Self::Class {
         Class::Stream(SuccessOrFailure::Failure, format!("{}", err))
     }
+
+    fn no_body(&self) -> Option<Self::Class> {
```

Comment:
How is this different from `eos(None)`?  Why does this not consume `self`?

Reply:
This is meant to try to classify before the body is streamed.